Python/Network_scanner/readme.md

```diff
@@ -0,0 +1,32 @@
+
+# Network Scanner in Python
+Python script that can scan all networks connected to same router and give's thier respectieve IP and MAC address.
+
+# Packages Used
+1. Scapy
+2. Optparse
+
+# How to run
+**1.** Fork [this](https://github.com/HarshCasper/Rotten-Scripts/) repository.
+Click on the <a href="https://github.com/HarshCasper/Rotten-Scripts/"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+
+```bash
+git clone https://github.com/<your-github-username>/Rotten-Scripts
+```
```

Comment:
You can remove these details. Just highlight how the Users are gonna use it.

Reply:
@HarshCasper I have made the changes, please do a review.